10LessonTen/Checkpoint2/css/style.css

```diff
@@ -0,0 +1,129 @@
+body{
+  font-family: walter turncoat;
+}
+
+
+a{
+  text-decoration: none;
+
+}
+
+ul{
+  list-style-type: none;
+  display:flex;
+
+  }
+
+nav{
+  display:flex;
+  justify-content: space-between;
+}
+
+nav p{
+  font-size: 20px;
+  font-weight: bold;
+
+}
+
+
+li{
+  margin-right: 30px;
+  font-size: 20px;
+  color:black;
+}
+
+header{
+  background-image:url(https://scontent.xx.fbcdn.net/v/t1.0-9/13891817_289499371415232_230777946967839666_n.jpg?oh=1f42c8042d546591e09a5dcd4ce3918e&oe=58102E99);
+    height:500px;
+    display:flex;
+    flex-direction:column;
+    align-items:center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-attachment: fixed;
+
+ }
+header h1, header p{
+    font-family:walter turncoat;
+  }
+
+header h1{
+  font-size:100px;
+  color:red;
+  margin-bottom:30px;
+  font-weight: bold;
+  }
+
+header p{
+  color:white;
+  font-size: 30px;
+  font-weight: bolder;
+
+}
+
+#first{
+  display:flex;
+  margin-bottom:70px;
+  margin-top:20px;
+}
+
+#second{
+ display:flex;
+ margin-bottom: 40px;
+}
+
+#third{
+  display:flex;
+  justify-content:space-around;
+  margin-bottom: 60px;
+  background-color: lightgrey;
+  padding-top: 60px;
+  }
+
+.four{
+  height:150px;
+  width:150px;
+
+}
+
+.one{
+  width:700px;
+  height:300px;
+  border: 3px solid black;
+  }
+
+.two{
+  height:400px;
+  width:300px;
+
```

Comment:
You'll notice that the left image is distorted since it doesn't fit in a 4x3 aspect ratio. To alleviate this, either use images that are the same aspect ratio and so will scale the same, or set the height to be a constant value and the `width:auto;` and accept that the images won't be the same width (but will be same height, so they still fit in your container all right). Orrrr, accept image distortion :)